### ISSUE TITLE
v 0.0.2 ERROR - Platform badnest does not generate unique IDs

### ISSUE THREAD

**Issue by stast1** [mattsch/badnest#3]
Debug log:
v0.0.2
```
2020-05-26 15:43:44 INFO (MainThread) [custom_components.badnest.climate] Adding nest thermostat uuid: 15AA01AC34170KW2
2020-05-26 15:43:44 INFO (MainThread) [custom_components.badnest.climate] Adding nest thermostat uuid: 15AA01AC34170KW2
2020-05-26 15:43:45 ERROR (MainThread) [homeassistant.components.climate] Entity id already exists - ignoring: climate.thermostat. Platform badnest does not generate unique IDs
```
```
2020-05-26 15:43:54 INFO (MainThread) [custom_components.badnest.sensor] Adding temperature sensors
2020-05-26 15:43:54 INFO (MainThread) [custom_components.badnest.sensor] Adding nest temp sensor uuid: 15AA01AC34170KW2
2020-05-26 15:43:54 INFO (MainThread) [custom_components.badnest.sensor] Adding nest temp sensor uuid: 15AA01AC34170KW2
2020-05-26 15:43:54 INFO (MainThread) [custom_components.badnest.sensor] Adding protect sensors
2020-05-26 15:43:54 ERROR (MainThread) [homeassistant.components.sensor] Entity id already exists - ignoring: sensor.thermostat_temperature_raw. Platform badnest does not generate unique IDs
```
Twice 

> Adding

v0.0.1
```
2020-05-26 15:54:06 INFO (MainThread) [custom_components.badnest.climate] Adding thermostats
2020-05-26 15:54:06 INFO (MainThread) [custom_components.badnest.climate] Adding nest thermostat uuid: 15AA01AC34170KW2
```
```
2020-05-26 15:54:18 INFO (MainThread) [custom_components.badnest.sensor] Adding temperature sensors
2020-05-26 15:54:18 INFO (MainThread) [custom_components.badnest.sensor] Adding nest temp sensor uuid: 15AA01AC34170KW2
2020-05-26 15:54:18 INFO (MainThread) [custom_components.badnest.sensor] Adding protect sensors
```

**Comment by mattsch** [comment]
I was able to reproduce with with the Protect sensors and the fix should be in v0.0.3.

**Comment by stast1** [comment]
Yes, now there are no errors. Thank!
